.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: Test Default Branch
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
Just fyi, as you can't know, but the current basf2 externals still use python 3.6.10, so the code should work with that, but I don't think it's an issue, I just [checked](https://docs.python.org/3.7/whatsnew/3.7.html)  there is no new syntax in 3.7 (walrus only appears in 3.8)

Reply:
We can also check with both 3.6 and 3.7.
Let's do this in another PR (at a later point in time)